media-proxy/src/rdma_session.c

```diff
@@ -359,55 +472,60 @@ static void *rx_rdma_frame_thread(void *arg)
 rx_rdma_session_context_t *rdma_rx_session_create(libfabric_ctx *dev_handle, rdma_s_ops_t *opts,
```

Comment:
Input params in rdma_rx_session_create() are not checked if for NULL

Reply:
fixed